internal/config/vsa.go

```diff
@@ -36,21 +36,28 @@ func NewVsaCommandOptions() *VsaCommandOptions {
 	}
 
 	c.fs = flag.NewFlagSet("vsa", flag.ContinueOnError)
-	c.fs.StringVar(&c.PolicyVersion, "policy-version", "", "GitHub release version of OPA bundle")
+	c.fs.Func("policy-url", "URL to retrieve policy bundle. Absolute paths will be handled as HTTP requests. Relative paths will be handled as local filepaths.", func(s string) error {
```

Comment:
I think this would be clearer if it said that paths with a scheme will be treated as HTTP/S. 

When I initially read this, I interpreted it to mean that something like `/home/user/policy.tgz` would be interpreted as a URL.

Reply:
I was going off the `IsAbs` wording, I'll make it more clear.